docs/themes/oso-tailwind/layouts/partials/header.html

```diff
@@ -68,8 +68,8 @@
                 </button>
             </a>
             {{ $downloadUrl := $.Site.GetPage "/reference/lib" }}
-            {{ if eq .Section "new-roles" }}
-            {{ $downloadUrl = $.Site.GetPage "/new-roles/install" }}
+            {{ if eq .Section "roles" }}
+            {{ $downloadUrl = $.Site.GetPage "/guides/roles/sqlalchemy/install" }}
```

Comment:
should this only link to `/guides/roles/sqlalchemy/install` if they're on a SQLAlchemy Roles page?